### PR TITLE
Update Mixer concept doc

### PR DESCRIPTION
**Pull request by douglas-reid**
This removes aspect stuff and attempts to bring in the config overview inline with current status. It also strips any mention of scripting from the site.

### CODE REVIEW

File: _docs/concepts/policy-and-control/mixer.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ streams intended for both the service operator as well as for service consumers.
 
 These mechanisms are applied based on a set of [attributes](./attributes.html) that are
 materialized for every request into Mixer. Within Istio, Envoy depends heavily on Mixer. Services running within the mesh
-can also use Mixer to report telemetry or manage quotas. (Note: as of Istio {{ site.data.istio.version }}, only Envoy can call Mixer.)
+can also use Mixer to report telemetry or manage quotas. (Note: as of Istio {{ site.data.istio.version }}, Mixer's API is meant solely for proxy (example: Envoy) use.)
```
Comment (rshriram) [inline _docs/concepts/policy-and-control/mixer.md]: the Mixer to .., meant for consumption solely by the Istio sidecar.

File: _docs/concepts/policy-and-control/mixer.md [hunk 2]
```diff
@@ -63,19 +63,13 @@ to target new or custom infrastructure backends.
 
 ## Configuration state
 
-Mixer's core runtime methods (`Check`, `Report`, and `Quota`) all accept a set of attributes on input and
-produce a set of attributes on output. The work that the individual methods perform is dictated by the set of input
-attributes, as well as by Mixer's current configuration. To that end, the service operator is responsible
-for:
+Mixer's core runtime methods ([Check](https://github.com/istio/api/blob/master/mixer/v1/service.proto#L43) and [Report](https://github.com/istio/api/blob/master/mixer/v1/service.proto#L48)) accept a set of attributes on input. Mixer's current configuration dictates the work that the individual methods perform with the set of input attributes. To that end, the service operator is responsible for:
```
Comment (rshriram) [inline _docs/concepts/policy-and-control/mixer.md]: Skip the links to code. Or potentially, point to links from references section

File: _docs/concepts/policy-and-control/mixer.md [hunk 3]
```diff
@@ -63,19 +63,13 @@ to target new or custom infrastructure backends.
...
 
-- Configuring the set of *aspects* that the deployment uses. An aspect is essentially a chunk of configuration
-state that configures an adapter (adapters being binary plugins as described [below](#adapters)).
+- Configuring a set of *handlers* for Mixer-generated data. Handlers are configured adapters (adapters being binary plugins as described [below](#adapters)). An example of handler configuration would be providing a statsd adapter with an IP address for a statsd backend.
 
-- Establishing the types of adapter parameters that Mixer can manipulate. These
-types are described in configuration through a set of *descriptors* (as described [here](./mixer-config#descriptors))
+- Configuring a set of *instances* for Mixer to generate based on attributes and literal values. Example of instances are _metric values_ and _log entries_. They represent a chunk of data that adapter code will operate on. For instance, an operator may configure Mixer to generate `request_count` metric values from attributes such as `destination.service` and `response.code`.
```
Comment (rshriram) [inline _docs/concepts/policy-and-control/mixer.md]: For example, an operator.. 
(Else, "for instance" gets confusing under _instances_)

File: _docs/concepts/policy-and-control/mixer.md [hunk 4]
```diff
@@ -63,19 +63,13 @@ to target new or custom infrastructure backends.
...
 
-- Creating rules to map the attributes of every incoming request into a 
-specific set of aspects and adapter parameters.
+- Configuring a set of *actions* that provide Mixer with the when, what, how, and where of instance handling. Actions dictate under what conditions instances should be generated, what instances to generated for those conditions and how they should be generated, and which handler should be sent the generated instances. These actions are grouped into *rules*. For instance, an action might tell Mixer to send generated `request_count` metric values to a statsd handler for all Report calls.
```
Comment (rshriram) [inline _docs/concepts/policy-and-control/mixer.md]: the mixer

Comment (rshriram) [reply _docs/concepts/policy-and-control/mixer.md]: Actions define the conditions under which instances should be generated, the set of instances to generate, how the instances should be generated, and the handlers that should process the generated instances.

Comment (rshriram) [reply _docs/concepts/policy-and-control/mixer.md]: "action might tell Mixer" => the Mixer

Comment (rshriram) [reply _docs/concepts/policy-and-control/mixer.md]: send generated `request_count` metric values -> send generated `request_count` metric instances ?

Comment (rshriram) [reply _docs/concepts/policy-and-control/mixer.md]: Over all, just as feedback: the first sentence is too abstract, and tad bit philosophical. If you are going to use an example, might be better to relate to each of the when, what, how and where explicitly.

File: _docs/concepts/policy-and-control/mixer.md [hunk 5]
```diff
@@ -86,9 +80,7 @@ Refer [here](./mixer-config.html) for detailed information on Mixer's configurat
 
 When a request comes in to Mixer, it goes through a number of distinct handling phases:
 
-- **Supplementary Attribute Production**. The first thing that happens in Mixer is to run a globally configured
-set of adapters that are responsible for introducing new attributes. These attributes are combined with the attributes
-from the request to form the total set of attributes for the operation.
+- **Supplementary Attribute Production**. The first thing that happens in Mixer is to run a globally configured set of adapters that are responsible for introducing new attributes. These attributes are combined with the attributes from the request to form the total set of attributes for the operation.
```
Comment (rshriram) [inline _docs/concepts/policy-and-control/mixer.md]: the Mixer

Comment (rshriram) [reply _docs/concepts/policy-and-control/mixer.md]: Better phrased as "The Mixer initially runs a globally configured set of adapters that are responsible for introducing new attributes".

Comment (douglas-reid) [reply _docs/concepts/policy-and-control/mixer.md]: Updated.

File: _docs/concepts/policy-and-control/mixer.md (hunk 4 above)
Comment (douglas-reid) [reply _docs/concepts/policy-and-control/mixer.md]: I've tried to simplify this a bit and reduce the abstract. Hopefully, I've improved it.

File: _docs/concepts/policy-and-control/mixer.md (hunk 3 above)
Comment (douglas-reid) [reply _docs/concepts/policy-and-control/mixer.md]: updated.

File: _docs/concepts/policy-and-control/mixer.md (hunk 2 above)
Comment (douglas-reid) [reply _docs/concepts/policy-and-control/mixer.md]: Removed links to code.

File: _docs/concepts/policy-and-control/mixer.md (hunk 1 above)
Comment (douglas-reid) [reply _docs/concepts/policy-and-control/mixer.md]: I've updated this to just remove any mention of services calling into Mixer. I think that just muddles the water for the overview.

**Comment by douglas-reid** [comment]
@rshriram thanks for the good review! PTAL - I've tried to address all of your comments. 

The only thing I didn't update was the switch to `the Mixer` instead of `Mixer`. Our style guide suggests that Mixer should be a proper noun. See: https://istio.io/v-0.1/docs/reference/contribute/style-guide.html#mixer.

File: _docs/concepts/policy-and-control/mixer.md (hunk 1 above)
Comment (rshriram) [reply _docs/concepts/policy-and-control/mixer.md]: proxy to Istio sidecar, as per our style guide.

**Pull request merged**